app/models/item.rb

```diff
@@ -35,6 +35,7 @@ def default_image
 
   def self.top_5
     Item.joins(:item_orders)
+        .where(active?: true)
```

Comment:
Making sure we are only getting top_5 data from active items

Reply:
Beautiful